src/components/Gnb.tsx

```diff
@@ -24,8 +24,8 @@ export const Gnb = () => {
   }, [push])
 
   const onClickMyPage = useCallback(() => {
-    push('/my-page')
```

Comment:
what is the intended behavior when `cid` is not defined (e.g., clicking my page at the class selection page)?

Reply:
why do you think having separate pages for classes is better than having a single page with a filter for classes?

Reply:
> what is the intended behavior when `cid` is not defined (e.g., clicking my page at the class selection page)?

I was thinking of making the option to go to myPage only available on the navbar once the class is selected:)

> why do you think having separate pages for classes is better than having a single page with a filter for classes?

I though that it would be redundant to choose class twice since the user already chose a class in the choose classes page.

Should I update it so that there is a drop down menu in my page and classes will be selected there and then a topic drop down menu will appear to select topics for that class 🤔?